README.md

```diff
@@ -69,10 +80,16 @@ $ yarn add fabric
 ```
 
 ```js
-// es6 imports
+// v6 es
+import * as fabric from 'fabric'; // browser
+import * as fabric from 'fabric/node'; // node
+// v6 cjs
+const fabric = require('fabric'); // browser
+const fabric = require('fabric/node'); // node
```

Comment:
so node14 supports import and so do most of the bundlers.
What is the reason we want to keep require in the documentation?

Reply:
I don't mind removing it

Reply:
done